docs/podman-build.1.md

```diff
@@ -39,15 +45,25 @@ instructions read from the Dockerfiles in the same way that environment
 variables are, but which will not be added to environment variable list in the
 resulting image's configuration.
 
+**--cache-from**
+
+Images to utilise as potential cache sources. Buildah does not currently support caching so this is a NOOP.
+
 **--cert-dir** *path*
 
-Use certificates at *path* (*.crt, *.cert, *.key) to connect to the registry.
+Use certificates at *path* (\*.crt, \*.cert, \*.key) to connect to the registry.
 Default certificates directory is _/etc/containers/certs.d_.
 
 **--cgroup-parent**=""
 
 Path to cgroups under which the cgroup for the container will be created. If the path is not absolute, the path is considered to be relative to the cgroups path of the init process. Cgroups will be created if they do not already exist.
 
+**--compress**
+
+This option is added to be aligned with other containers CLIs.
+Buildah doesn't communicate with a daemon or a remote server.
```

Comment:
Should we change Buildah to Podman throughout  and/or explain that Buildah is being used by Podman to do the builds?

Reply:
Yes, cut and paste.

Reply:
Fixed